media-sound/sox/sox-14.4.2.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A command line utility to do audio format conversions"
+DESCRIPTION="
+SoX is a cross-platform (Windows, Linux, MacOS X, etc.) command line utility \
+that can convert various formats of computer audio files in to other formats. \
+It can also apply various effects to these sound files, and, as an added \
+bonus, SoX can play and record audio files on most platforms 
+"
+HOMEPAGE="http://sox.sourceforge.net"
+SOURCE_URI="http://sourceforge.net/projects/sox/files/sox/14.4.2/sox-14.4.2.tar.bz2"
+CHECKSUM_SHA256="81a6956d4330e75b5827316e44ae381e6f1e8928003c6aa45896da9041ea149c"
+LICENSE="GNU GPL v2"
+COPYRIGHT="1991-2015 Lance Norskog"
+REVISION="1"
+ARCHITECTURES="x86 !x86_gcc2 ?x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	sox$secondaryArchSuffix = $portVersion compat >= 1
+	cmd:sox$secondaryArchSuffix = $portVersion
+	cmd:play$secondaryArchSuffix = $portVersion
+	cmd:rec$secondaryArchSuffix = $portVersion
+	cmd:soxi$secondaryArchSuffix = $portVersion
+	lib:libsox$secondaryArchSuffix = 0.0.0 compat >= 0
+"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libflac$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libmp3lame$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libflac$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libmp3lame$secondaryArchSuffix
+	devel:libsndfile$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+"
+
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:autoconf
+"
+
+PATCHES="sox-14.4.2.patchset"
+
+SOURCE_DIR="sox-14.4.2"
+
+BUILD()
+{
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	
+	prepareInstalledDevelLibs libsox
+	fixPkgconfig
+	
+	packageEntries devel $developDir
+}
+
+PROVIDES_devel="
+	sox${secondaryArchSuffix}_devel = $portVersion
+	devel:libsox$secondaryArchSuffix = 0.0.0 compat >= 0
+"
+
+REQUIRES_devel="
+	sox${secondaryArchSuffix} == $portVersion base
+"
```

Comment:
Put PROVIDES_devel/REQUIRES_devel between PROVIDES/REQUIRES and BUILD_REQUIRES/PREREQUIRES.